commands/browsing/convert-twitter-to-nitter.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+// Required parameters:
+// @raycast.schemaVersion 1
+// @raycast.title Convert Twitter to Nitter
+// @raycast.mode silent
+
+// Optional parameters:
+// @raycast.icon 🐔
+
+// Documentation:
+// @raycast.description Convert Twitter link to Nitter
+// @raycast.author csharp
+// @raycast.authorURL https://raycast.com/csharp
```

Comment:
According to your GitHub handle, you are @noidwasavailable, why not point to your profile in here? 😉
```suggestion
// @raycast.author cSharp
// @raycast.authorURL https://github.com/noidwasavailable
```